database/seeds/UsersTableSeeder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class UsersTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        \DB::table('users')->insert([
+            'name' => 'Sugawara Ayaka',
+            'email' => 'Ayaka@gmai.com',
+            'password' => Hash::make('ayaka0000')
+        ]);
+        \DB::table('users')->insert([
+            'name' => 'Yamada tarou',
+            'email' => 'yamada@gmai.com',
+            'password' => Hash::make('yamada0000'),
+        ]);
+        \DB::table('users')->insert([
+            'name' => 'Hanako',
+            'email' => 'Hanako@gmai.com',
+            'password' => Hash::make('Hanako0000'),
+        ]);
```

Comment:
（参考までに）以下のように記述すれば、複数回、テーブル名を記述する必要はないかもです！

参考：https://harusamelab.com/2020/03/31/laravel%E3%81%A7db%E3%81%AB%E5%88%9D%E6%9C%9F%E3%83%87%E3%83%BC%E3%82%BF%E3%82%92%E8%A8%AD%E5%AE%9A%E3%81%99%E3%82%8Bseeder%E3%81%AE%E4%BD%9C%E6%88%90/

```suggestion
        DB::table('users')->insert([
            [
                'name' => 'Sugawara Ayaka',
                'email' => 'Ayaka@gmai.com',
                'password' => Hash::make('ayaka0000')
            ],
            [
                'name' => 'Yamada tarou',
                'email' => 'yamada@gmai.com',
                'password' => Hash::make('yamada0000')
            ],
            [
                'name' => 'Hanako',
                'email' => 'Hanako@gmai.com',
                'password' => Hash::make('Hanako0000')
            ]
        ]);
```

Reply:
お疲れ様です。ご指摘ありがとうございます！複数回、テーブル名を記述しなくて良い方法を知らなかった
ためとても参考になりました。教えて頂いた通りに変更させて頂きます。emailも特に理由なく大文字にしていたので変更させて頂きます。よろしくお願い致します。